exercises/eliuds-eggs/metadata.toml

```
@@ -0,0 +1,2 @@
+title = "Eliud's Eggs"
+blurb = "Count the 1 bits in a number"
```

I would add your name and vaeng's as `source` and the forum URL as `source_url`:

```suggestion
blurb = "Count the 1 bits in a number"
source = "[keiravillekode, vaeng"
source_url = "https://forum.exercism.org/t/new-exercise-suggestion-pop-count/7632/4"
```

We did something similar for `resistor-color`: https://github.com/exercism/problem-specifications/blob/main/exercises/resistor-color/metadata.toml

P.S. I don't know if you'd like to use your own name or you nickname as source (same goes for vaeng)

---

My real name is fine :)

---

source and source_url added